nucypher/blockchain/eth/sol/source/contracts/MinersEscrow.sol

```diff
@@ -1165,15 +1163,12 @@ contract MinersEscrow is Issuer {
         }
     }
 
-    function verifyState(address _testTarget) public onlyOwner {
+    function verifyState(address _testTarget) public {
         super.verifyState(_testTarget);
-        require(uint16(delegateGet(_testTarget, "minLockedPeriods()")) ==
-            minLockedPeriods);
-        require(delegateGet(_testTarget, "minAllowableLockedTokens()") ==
-            minAllowableLockedTokens);
-        require(delegateGet(_testTarget, "maxAllowableLockedTokens()") ==
-            maxAllowableLockedTokens);
-        require(address(uint160(delegateGet(_testTarget, "policyManager()"))) == address(policyManager));
```

Comment:
What happened to the type casting here - is this still part of an effort to stay up-to-date with the latest compilers?

Reply:
I just decided to remove explicit casting `uint256` -> `uint160` -> `address`, because `addres` is `uint160`. If you prefer more explicit casting - I'll revert it

Reply:
No - it's OK.  Just making sure I understand correctly.  Are the `address` and `uint160` literally the same type?

Reply:
Not literally, I said too generally, I mean `uint256` -> `uint160` and `uint256` -> `address` removes zeros from the beggining of the value, so `uint160` -> `address` did nothing